src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name } = props;
```

Comment:
[Optional] - You can use a shortcut for that one too and do `const Button = ({ name }) =>`